packages/amplify-migration-template-gen/src/resolvers/cfn-output-resolver.ts

```diff
@@ -59,6 +59,44 @@ class CfnOutputResolver {
       }
     });
 
+    const fnGetAttRegExp = new RegExp(`{"${GET_ATT}":\\["(?<LogicalResourceId>\\w+)","(?<AttributeName>\\w+)"]}`, 'g');
```

Comment:
we can extract the below logic of replacing with physicalIds to a separate method.